patches/chrome-android-expectations-lint-suppressions.xml.patch

```diff
@@ -6,5 +6,5 @@ index 814a84c9b838db7bbd52873344f366ed2f9a73e5..513b3889a4c9ae85af74515b74896909
      <ignore regexp="Must be one of: LineBreaker.BREAK_STRATEGY_SIMPLE, LineBreaker.BREAK_STRATEGY_HIGH_QUALITY, LineBreaker.BREAK_STRATEGY_BALANCED"/>
    </issue>
-+  <issue id="ExportedService" severity="ignore"/><issue id="DuplicateDefinition"><ignore regexp="chrome/android/java/res/values/brave_colors.xml"/></issue><issue id="StringFormatCount"><ignore regexp="sponsored_images_bottom_sheet_text"/><ignore regexp="notification_marketing"/><ignore regexp="renew_monthly_save"/><ignore regexp="wallet_swap_fees_notice"/><ignore regexp="processing_fee"/></issue><issue id="WrongConstant"><ignore regexp="ContentSettingsType.BRAVE_CONTENT_SETTINGS_TYPES_LIST"/></issue><issue id="UnusedResources"><ignore regexp="The resource `R.string.playlist_.*` appears to be unused"/><ignore regexp="The resource `R.string.new_tab_otr.*` appears to be unused"/><ignore regexp="The resource `R.dimen.md_incognito_ntp.*` appears to be unused"/><ignore regexp="The resource `R.layout.incognito.*` appears to be unused"/><ignore regexp="The resource `R.layout.logo_view_layout` appears to be unused"/><ignore regexp="The resource `R.style.*_BrowserUI_Switch_Incognito` appears to be unused"/><ignore regexp="The resource `R.style.CardTransparentForDark` appears to be unused"/><ignore regexp="The resource `R.dimen.toolbar_hairline_height` appears to be unused"/><ignore regexp="The resource `R.style.TextAppearance_IncognitoNewTabLearnMoreLinkModern` appears to be unused"/></issue><issue id="UnsafeOptInUsageError"><ignore regexp='(markerClass = androidx.media3.common.util.UnstableApi.class)'/></issue>
++  <issue id="ExportedService" severity="ignore"/><issue id="DuplicateDefinition"><ignore regexp="chrome/android/java/res/values/brave_colors.xml"/></issue><issue id="StringFormatCount"><ignore regexp="sponsored_images_bottom_sheet_text"/><ignore regexp="notification_marketing"/><ignore regexp="renew_monthly_save"/><ignore regexp="wallet_swap_fees_notice"/><ignore regexp="processing_fee"/></issue><issue id="WrongConstant"><ignore regexp="ContentSettingsType.BRAVE_CONTENT_SETTINGS_TYPES_LIST"/></issue><issue id="UnusedResources"><ignore regexp="The resource `R.string.new_tab_otr.*` appears to be unused"/><ignore regexp="The resource `R.dimen.md_incognito_ntp.*` appears to be unused"/><ignore regexp="The resource `R.layout.incognito.*` appears to be unused"/><ignore regexp="The resource `R.layout.logo_view_layout` appears to be unused"/><ignore regexp="The resource `R.style.*_BrowserUI_Switch_Incognito` appears to be unused"/><ignore regexp="The resource `R.style.CardTransparentForDark` appears to be unused"/><ignore regexp="The resource `R.dimen.toolbar_hairline_height` appears to be unused"/><ignore regexp="The resource `R.style.TextAppearance_IncognitoNewTabLearnMoreLinkModern` appears to be unused"/></issue><issue id="UnsafeOptInUsageError"><ignore regexp='(markerClass = androidx.media3.common.util.UnstableApi.class)'/></issue>
```

Comment:
what is being changed here?

Reply:
we needed to include the strings for translation into brave core so we had a lint suppression for kotlin module strings which is not required now. we had a patch for : `<ignore regexp="The resource `R.string.playlist_.*` appears to be unused"/>`, which was removed.